python_modules/libraries/dagster-looker/dagster_looker/api/assets.py

```diff
@@ -0,0 +1,83 @@
+from typing import Sequence, Type, cast
+
+from dagster import AssetExecutionContext, AssetsDefinition, Failure, multi_asset
+
+from dagster_looker.api.dagster_looker_api_translator import (
+    DagsterLookerApiTranslator,
+    LookerStructureData,
+    LookerStructureType,
+    LookmlView,
+    RequestStartPdtBuild,
+)
+from dagster_looker.api.resource import LookerResource
+
+
+def build_looker_pdt_assets_definitions(
+    resource_key: str,
+    request_start_pdt_builds: Sequence[RequestStartPdtBuild],
+    dagster_looker_translator: Type[DagsterLookerApiTranslator] = DagsterLookerApiTranslator,
+) -> Sequence[AssetsDefinition]:
+    """Returns the AssetsDefinitions of the executable assets for the given the list of refreshable PDTs.
+
+    Args:
+        resource_key (str): The resource key to use for the Tableau resource.
+        request_start_pdt_builds (Optional[Sequence[RequestStartPdtBuild]]): A list of requests to start PDT builds.
+            See https://developers.looker.com/api/explorer/4.0/types/DerivedTable/RequestStartPdtBuild?sdk=py
+            for documentation on all available fields.
+        dagster_looker_translator (Optional[DagsterLookerApiTranslator]): The translator to
+            use to convert Looker structures into assets. Defaults to DagsterLookerApiTranslator.
+
+    Returns:
+        AssetsDefinition: The AssetsDefinitions of the executable assets for the given the list of refreshable PDTs.
+    """
+    translator = dagster_looker_translator()
+    result = []
+    for request_start_pdt_build in request_start_pdt_builds:
+
+        @multi_asset(
+            specs=[
+                translator.get_asset_spec(
+                    LookerStructureData(
+                        structure_type=LookerStructureType.VIEW,
+                        data=LookmlView(
+                            view_name=request_start_pdt_build.view_name,
+                            sql_table_name=None,
+                        ),
+                    )
+                )
```

Comment:
We create the specs for PDTs in this function because it requires the list of `RequestStartPdtBuild` objects. We don't create asset specs for PDTs unless we add them as materializable assets.

The other solution would be to pass the list of `RequestStartPdtBuild` to the `load_looker_asset_specs` function, but this is a bit awkward and confusing because these are materializable assets.